packages/runtime/src/api/react.tsx

```diff
@@ -141,6 +144,12 @@ export function useMutation<
   return useApolloMutation(mutation, { client: client?.apolloClient, ...options })
 }
 
+export function useApolloClient() {
```

Comment:
This hook doesn't seem to be used anywhere.

Reply:
Removed